BUILDING.md

```diff
@@ -243,8 +243,8 @@ transition before the year-end deadline.
 * `gcc` and `g++` >= 6.3 or newer, or
 * GNU Make 3.81 or newer
 * Python (see note above)
-    * Python 2.7
-    * Python 3.5, 3.6, and 3.7 are experimental.
+  * Python 2.7
```

Comment:
I do not know about these, might need a different style more than anything.

Reply:
Not sure I follow the comment. Markdown is happy with 2 space indent for unordered lists (but not ordered lists) and this also matches the EditorConfig setting of 2 space indenting.

Reply:
ok